src/main/java/hudson/remoting/ChunkedOutputStream.java

```diff
@@ -94,4 +94,4 @@ private void sendFrame(boolean hasMore) throws IOException {
         base.write(buf,0,size);
         size = 0;
     }
-}
\ No newline at end of file
+}
```

Comment:
I'm trying to revert this but cant seem to get it to go back to the previous state - sorry about the non-relevant whitespace change.